Library/Homebrew/os/mac/xcode.rb

```diff
@@ -254,9 +254,8 @@ def detect_version
         # CLT isn't a distinct entity pre-4.3, and pkgutil doesn't exist
         # at all on Tiger, so just count it as installed if Xcode is installed
         return MacOS::Xcode.version if MacOS::Xcode.installed? && Xcode::Version.new(MacOS::Xcode.version) < "3.0"
-
         [MAVERICKS_PKG_ID, MAVERICKS_NEW_PKG_ID, STANDALONE_PKG_ID, FROM_XCODE_PKG_ID].find do |id|
-          if Xcode::Version.new(MacOS.version) >= :mavericks.to_s
+          if Xcode::Version.new(MacOS.version) >= Xcode::Version.new(MacOS::Version.from_symbol(:mavericks))
```

Comment:
It's done. I've updated the commit message and the title of this pull request as well but left the branch name as is.